CMSgov/design-system
# [WNMGDS-1455] removing visited link color from hgov menu items

## Summary

Updating hgov header links (in logged out & logged in variations) to maintain their primary color when the link has been visited.

Note, the selector in the CSS includes some `:not()`s because we want the link to change color when in an active or hover state.

## How to test

You can run `yarn storybook:healthcare` and navigate to components > Header and play around with the links.

Here's a gif where the Spanish link has the visited state enabled in browser tools and the login link does not. Notice how both links behavior similarly for focusing, clicking & hovering
![HgovHeaderLinks](https://user-images.githubusercontent.com/33579665/158271563-27770230-ab95-4570-ada8-8d4ef37e43fa.gif)
